src/Uno.UI.Toolkit/Web/CookieManager.cs

```diff
@@ -0,0 +1,93 @@
+﻿#if __WASM__
+#nullable enable
+
+using System;
+using System.Linq;
+using System.Text;
+using Uno.Foundation;
+
+namespace Uno.UI.Toolkit.Web
+{
+	public class CookieManager
```

Comment:
I don't think this class should be in the toolkit. I'd think it would make more sense to place in the `Uno.Web.Http` namespace to be consumer from there. If not possible, then at the same level as the `WasmHttpHandler`.

Reply:
I put it in Toolkit as a Uno specific functionality, but can be moved to Uno.UWP without issues 👍